api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -316,4 +316,22 @@ service ConfigMgmt {
   rpc NodeExport(cfgmgmt.request.NodeExport) returns (stream chef.automate.api.common.ExportData) {};
 
   rpc ReportExport(cfgmgmt.request.ReportExport) returns (stream chef.automate.api.common.ExportData) {};
+
+  /*
+  GetNodesFieldValueCounts
+
+  Returns a list fields with value counts
```

Comment:
nit: "list of fields" ?
maybe include an example for clarity?

Reply:
maybe also add a sentence about how the counts are affected by the filters? (e.g. counts respect all filters except their own or some other wording that makes sense?)
```
[19][default:/src:130]# curl -s -f --insecure -H "api-token: $(get_api_token)" "https://localhost/api/v0/cfgmgmt/nodes_field_value_counts?terms=platform&terms=status" | jq
{
  "fields": [
    {
      "terms": [
        {
          "term": "mac_os_x 10.11.5",
          "count": 28
        },
        {
          "term": "linux 8.9",
          "count": 1
        },
        {
          "term": "macos 8.9",
          "count": 1
        },
        {
          "term": "windows 8.9",
          "count": 1
        }
      ],
      "field": "platform"
    },
    {
      "terms": [
        {
          "term": "missing",
          "count": 29
        },
        {
          "term": "failure",
          "count": 2
        }
      ],
      "field": "status"
    }
  ]
}
[20][default:/src:0]# curl -s -f --insecure -H "api-token: $(get_api_token)" "https://localhost/api/v0/cfgmgmt/nodes_field_value_counts?terms=platform&terms=status&filter=platform:linux*" | jq
{
  "fields": [
    {
      "terms": [
        {
          "term": "mac_os_x 10.11.5",
          "count": 28
        },
        {
          "term": "linux 8.9",
          "count": 1
        },
        {
          "term": "macos 8.9",
          "count": 1
        },
        {
          "term": "windows 8.9",
          "count": 1
        }
      ],
      "field": "platform"
    },
    {
      "terms": [
        {
          "term": "missing",
          "count": 1
        }
      ],
      "field": "status"
    }
  ]
}
```